### PR TITLE
Add image and location fields when creating event

### PR DESCRIPTION
**Pull request by martinothamar**
Had to do this for the demo last week

### CODE REVIEW

File: src/pages/LandingPage/components/EventList/styled.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const GridItem = styled.li`
   }
 
   @media screen and (min-width: 1250px) {
-    width: 29vw;
+    width: 28vw;
```
Comment (martinothamar) [inline src/pages/LandingPage/components/EventList/styled.ts:21]: This isn't very scientific, but at `29` layout wasn't correct, but `28` looked OK for the demo

Comment (anhtin) [reply src/pages/LandingPage/components/EventList/styled.ts:21]: Typical 😅

File: src/pages/EventPage/components/MetaInfo/MetaInfo.tsx [hunk 2]
```diff
@@ -6,13 +6,14 @@ import { Event } from 'src/types/domain';
...
   Event,
-  'name' | 'startTime' | 'endTime' | 'location' | 'owner'
+  'name' | 'startTime' | 'endTime' | 'location' | 'owner' | 'imageUrl'
 >;
 
 function MetaInfo({
   name,
   startTime,
   endTime,
+  imageUrl,
```
Comment (anhtin) [inline src/pages/EventPage/components/MetaInfo/MetaInfo.tsx:16]: Is there any point in adding this if we're not using it yet?

**Review by anhtin — APPROVED** [review]
Ah damnit, sorry this had to happen on the demo :/

File: src/pages/EventPage/components/MetaInfo/MetaInfo.tsx (hunk 2 above)
Comment (martinothamar) [reply src/pages/EventPage/components/MetaInfo/MetaInfo.tsx:16]: No

**Comment by anhtin** [comment]
I'm merging this.

**Pull request merged**